lib/utils.pm

```diff
@@ -150,13 +150,13 @@ sub integration_services_check {
         my $vmname       = console('svirt')->name;
         my $ips_host_pov = console('svirt')
           ->get_cmd_output('powershell -Command "Get-VM ' . $vmname . ' | Get-VMNetworkAdapter | Format-Table -HideTableHeaders IPAddresses"');
-        $ips_host_pov = (split /\n/, $ips_host_pov)[1];
-        $ips_host_pov =~ s/,.*//g;
-        $ips_host_pov =~ s/\{//g;
-        my $ips_guest_pov = script_output("ip address show up scope global | grep -w inet | awk '{ print \$2 }' | sed 's|/.*||' | tr -d '\\n'");
+        $ips_host_pov =~ m/(\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3})/;
```

Comment:
Just for fun, here is more precise regexp for the ip v4 address =) (if I haven't confused anything):
```
  ((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}
  (25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)
```